.github/CODEOWNERS

```diff
@@ -605,7 +605,6 @@
 /pkg/ebpf/map_cleaner*.go               @DataDog/universal-service-monitoring
 /pkg/compliance/                        @DataDog/agent-cspm
 /pkg/databasemonitoring                 @DataDog/database-monitoring
-/pkg/datastreams                        @DataDog/data-streams-monitoring
```

Comment:
Can you add your team as owners or co-owners of `comp/core/autodiscovery/providers/datastreams/`?